Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/Extensions/ClaimSetResourceClaimActionAuthStrategiesExtension.cs

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.Ods.AdminApi.Infrastructure.ClaimSetEditor;
+using EdFi.Ods.AdminApi.Infrastructure.ClaimSetEditor.Extensions;
+using Action = EdFi.Ods.AdminApi.Infrastructure.ClaimSetEditor.Action;
+
+namespace EdFi.Ods.AdminApi.Infrastructure.Services.ClaimSetEditor.Extensions;
+
+public static class ClaimSetResourceClaimActionAuthStrategiesExtension
+{
+    public static ClaimSetResourceClaimActionAuthStrategies? Create(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies)
+    {
+        return claimSetResourceClaimActionAuthStrategies[0];
+    }
+
+    public static ClaimSetResourceClaimActionAuthStrategies? Read(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies)
+    {
+        return claimSetResourceClaimActionAuthStrategies[1];
+    }
+
+    public static ClaimSetResourceClaimActionAuthStrategies? Update(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies)
+    {
+        return claimSetResourceClaimActionAuthStrategies[2];
+    }
+
+    public static ClaimSetResourceClaimActionAuthStrategies? Delete(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies)
+    {
+        return claimSetResourceClaimActionAuthStrategies[3];
+    }
+
+    public static ClaimSetResourceClaimActionAuthStrategies? ReadChanges(this ClaimSetResourceClaimActionAuthStrategies?[] claimSetResourceClaimActionAuthStrategies)
+    {
+        return claimSetResourceClaimActionAuthStrategies[4];

```

Comment:
@jagudelo-gap 
Please check the array length is greater then the index value. If array is smaller than the index value will cause "Index out of range" error.